.githooks/pre-commit

```diff
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
```

Comment:
Any reason for that?
I think it would be better to use default shell in the system, unless we need some bashisms.

Reply:
when I started developing this PR it had one, but I see it was already corrected, I'll revert this to use sh